CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Features
 
+### ⚠ BREAKING CHANGES
+
+* **core, platform:** angular 13 migration
```

Comment:
beside angular 13, are there other changes expected from the teams using ngx?

Reply:
no, only migration